dense_reconstruction/ViewInformationRequest call
---
dense_reconstruction/ViewInformation expected_information